plugin-src/transformers/partials/transformMaskChildren.ts

```diff
@@ -0,0 +1,43 @@
+import { transformGroupNodeLike, transformSceneNode } from '@plugin/transformers';
+
+import { PenpotNode } from '@ui/lib/types/penpotNode';
+import { GroupShape } from '@ui/lib/types/shapes/groupShape';
+
+export const transformMaskChildren = async (
+  children: readonly SceneNode[],
+  baseX: number,
+  baseY: number
+): Promise<PenpotNode[]> => {
+  const splitChildren: PenpotNode[] = [];
+  let currentGroup: Partial<GroupShape> | null = null;
+
+  for (const child of children) {
+    const transformedChild = await transformSceneNode(child, baseX, baseY);
+    if (!transformedChild) {
+      continue;
+    }
+
+    if ('isMask' in child && child.isMask) {
```

Comment:
Maybe extract this to a function? What kind of objects in Figma have the isMask property?